src/main/scala/edu/berkeley/ce/rockslicing/Delaunay.scala

```diff
@@ -23,7 +23,7 @@ object Delaunay {
     case class EdgeAnihilationSet(s: Set[IEDGE]) {
       def add(e: IEDGE): EdgeAnihilationSet = {
         if( s.contains(e) ) {
-          printf(s"FOUND ${e} NOT REVERSED *********************\n")
+//          printf(s"FOUND ${e} NOT REVERSED *********************\n")
```

Comment:
Is there any particular reason we'd ever want to see this output? Just want to be sure we aren't setting ourselves up for any confusion later by disabling this printing.